include/macros.h

```diff
@@ -108,6 +108,11 @@
 #define SET_STOLEN_ITEM_2(itemId) \
     (gSaveContext.save.stolenItems = (gSaveContext.save.stolenItems & ~0x00FF0000) | ((itemId & 0xFF) << 0x10))
 
+#define GET_TOWN_SHOOTING_GALLERY_HIGH_SCORE() ((s32)(gSaveContext.save.shootingGalleryHighScores & 0xFFFF))
+#define GET_SWAMP_SHOOTING_GALLERY_HIGH_SCORE() ((s32)((gSaveContext.save.shootingGalleryHighScores & 0xFFFF0000) >> 0x10))
+#define SET_TOWN_SHOOTING_GALLERY_HIGH_SCORE(score) (gSaveContext.save.shootingGalleryHighScores = (gSaveContext.save.shootingGalleryHighScores & 0xFFFF0000) | ((u16)(score)))
+#define SET_SWAMP_SHOOTING_GALLERY_HIGH_SCORE(score) (gSaveContext.save.shootingGalleryHighScores = ((gSaveContext.save.shootingGalleryHighScores) & 0xFFFF) | ((u16)(score) << 0x10))
+
```

Comment:
https://github.com/zeldaret/mm/pull/894#discussion_r913281446 I think it makes more sense in macros.h for now personally.

Reply:
I don't, there's a lot of stuff in macros.h that shouldn't be already, this is about accessing specific stuff from savecontext so it makes more sense to me to have it there than in a general dumping file.

Reply:
Many of our macros in `macros.h` are about accessing specific stuff from savecontext (the stolen item macros even have the same general structure), so I still think it is best to keep them grouped together for the time being, and I'd prefer to bloat `macros.h` than `z64save.h`.

Reply:
Personally I would prefer to move the macros to access `gSaveContext` in `z64save.h`, but in that case we should probably move the ones from macros.h to that header too and I can understand if you don't want to do that right now because that would be a bit out of the scope of this PR.
I should probably have done that cleanup on my sram PR... sorry :sweat_smile:

Reply:
I'll leave it up to darkeye (it is his PR after all).

Reply:
Sure, I can move everything that accesses `gSaveContext` to `z64save.h`.